frontend/packages/core/src/Input/text-field.tsx

```diff
@@ -91,6 +90,13 @@ const StyledTextField = styled(BaseTextField)({
   },
 });
 
+const IconButton = styled(MuiIconButton)({
+  borderRadius: "0",
+  backgroundColor: "#E7E7EA",
+  borderBottomRightRadius: "3px",
+  borderTopRightRadius: "3px",
+});
```

Comment:
what about focus, hover, etc?

Reply:
👍  I was going to follow up with the implementation of icon only buttons but I can add those states here and extract later